lib/rentlinx/client.rb

```diff
@@ -91,7 +110,8 @@ def response_handler(response)
       when 204
         nil # don't attempt to JSON parse emptystring
       when 400
-        raise Rentlinx::BadRequest, response
+        body = JSON.parse(response.body)
```

Comment:
> If you reference other comments, please provide links to them, particularly if the comments are on another PR. Thanks :)

Normally I would, unfortunately I am less capable of finding the old commit hash then you are. I don't get email notifications for my own comments :(